building.html

```diff
@@ -0,0 +1,193 @@
+
+      <ol>
+        <li>
+          <p>Clone the repository from GitHub, including all submodules, using a git client. On the command line, this can be done by executing the following:<br>
+            <code>git clone --recurse-submodules https://github.com/H-uru/Plasma.git</code></p>
+
+          <p class="infonote" data-infotype="warning"><b>Note:</b> We recommend using <a href="https://git-scm.com/download/win">Git for Windows</a>. Using the git program provided by Cygwin will not set permissions properly on some files and result in build failures.</p>
+
+          <p class="infonote" data-infotype="warning"><b>Note:</b> Some dependencies may fail to build if the path to the code contains spaces.</p>
+        </li>
+
+        <li>
+          <p>Launch Visual Studio.</p>
+        </li>
+
+        <li>
+          <p>Select <strong>Open a local folder</strong> and choose the folder where you cloned the repository.</p>
+        </li>
+
+        <li>
+          <p>Compile the project by using <strong>Build &gt; Install Plasma</strong>. All required dependencies will automatically be downloaded from vcpkg as part of the build process.</p>
+
+          <details id="building-windows-32">
+            <summary class="expanding-heading">Building a 32-bit client on a 64-bit system</summary>
+
+            <ol>
+              <li>
+                <p>Wait for Visual Studio to finish preparing the workspace.<br>Look for a line that says “<tt>CMake generation finished</tt>” in the Output window.</p>
+              </li>
+
+              <li>
+                <p>Open the CMake Settings (<strong>Project &gt; CMake Settings for Plasma</strong>).</p>
+              </li>
+
+              <li>
+                <p>Click the green <strong>+</strong> button to add a new configuration.</p>
+              </li>
+
+              <li>
+                <p>Select “x86-Release” from the popup window.</p>
+              </li>
+
+              <li>
+                <p>Save the CMake settings, using Ctrl+S or the Save button in the toolbar.</p>
+              </li>
+
+              <li>
+                <p>In the dropdown menu in the toolbar, select the new “x86-Release” configuration.</p>
+              </li>
+
+              <li>
+                <p>Compile the project by using <strong>Build &gt; Install Plasma</strong>.</p>
+              </li>
+            </ol>
+          </details>
+        </li>
+
+        <li>
+          <p>The client will be built and installed into a subfolder of where you cloned the repo.</p>
+
+          <dl class="wider">
+            <dt>32-bit Debug:</dt>
+            <dd><tt>out\install\x86-Debug\client</tt></dd>
+
+            <dt>32-bit Release:</dt>
+            <dd><tt>out\install\x86-Release\client</tt></dd>
+
+            <dt>64-bit Debug:</dt>
+            <dd><tt>out\install\x64-Debug\client</tt></dd>
+
+            <dt>64-bit Release:</dt>
+            <dd><tt>out\install\x64-Release\client</tt></dd>
+          </dl>
```

Comment:
I think this section needs more detail. When I initially saw it, my reaction was: "wait, what are we talking about?" There's no mention of arch or build types before now by default.

Reply:
I don't know how to tell people where to find the files when the output folder varies, without also mentioning the fact that it varies 😛